src/renderer/viz/expressions/classification/ViewportStandardDev.js

```diff
@@ -58,7 +58,7 @@ export default class ViewportStandardDev extends Classifier {
             throw new RangeError(`The 'classSize' must be > 0.0, but '${classSize}' was used.`);
         }
 
-        const children = { input, _histogram: viewportHistogram(input) };
+        const children = { input, _histogram: viewportHistogram(input, 1000) };
```

Comment:
Maybe we could add an explaining constant here. 
In the future we could even consider exposing this as a ViewportStandardDev parameter, using 1000 as default value

Reply:
To be honest, I changed it because tests were failing. I guess we were relying on the distribution over 1000 bins.

Reply:
Even for the stdDev calculation?

Reply:
I don't know what do you mean about `stdDev calculation` :confused:

Reply:
I was just wondering about the effect of having a lower number in the precision of the calculation made at `_standardDevFrom(histogram, average)` function

Reply:
ah, yes, but we can/should override the default for that, right?

Reply:
👍 That's it. We should add it as an external parameter and then choose a good default